coalib/bearlib/abstractions/ExternalBearWrap.py

```diff
@@ -0,0 +1,174 @@
+import json
+from functools import partial
+from collections import OrderedDict
+
+from coalib.bears.LocalBear import LocalBear
+from coala_decorators.decorators import enforce_signature
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.FunctionMetadata import FunctionMetadata
+
+
+def _prepare_options(options):
+    """
+    Prepares options for a given options dict in-place.
```

Comment:
This function is similar to `_prepare_options` in linter. I figured I would let it the same but I didn't need to modify the options in the end. I will modify the docstring to be more relevant to what the function does.